package.json

```diff
@@ -85,7 +84,6 @@
     "@types/cli-spinner": "^0.2.0",
     "@types/crc": "^3.4.0",
     "@types/debug": "4.1.12",
-    "@types/file-type": "^10.9.1",
```

Comment:
I removed this because during `npm i` it was showing a warning that it was just a stub and should be removed (and we use `file-type` @ 16.x atm)